Bottom: 7fbc784798e1c807824992ca78ca43155a8647a8
Top:    32bf53139fbe620aee0e9a45ffd6b9e46f950459
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 14:01:38 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index c138b71..412c9c8 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -885,8 +885,7 @@ static int validate_location(unsigned int logopt, char *loc)
 		    !strncmp(ptr, "file:", 5) || !strncmp(ptr, "yp:", 3) ||
 		    !strncmp(ptr, "nis:", 4) || !strncmp(ptr, "nisplus:", 8) ||
 		    !strncmp(ptr, "ldap:", 5) || !strncmp(ptr, "ldaps:", 6) ||
-		    !strncmp(ptr, "sss:", 4) || !strncmp(ptr, "dir:", 4) ||
-		    !strncmp(ptr, "hosts:", 4))
+		    !strncmp(ptr, "sss:", 4) || !strncmp(ptr, "dir:", 4))
 			return 1;
 		error(logopt,
 		      "expected colon delimeter not found in location %s",
